### DISCUSSION TITLE
部署和asgi相关问题

### DISCUSSION THREAD

**Discussion by lsm1103** [fastapi/fastapi#7260] [Questions]
![20200825101728](https://user-images.githubusercontent.com/37491314/91118473-a20d1900-e6c3-11ea-931e-5864c9e038b2.jpg)

会有内存溢出的问题，关键嵌套的递归还不多的情况下

![20200825101753](https://user-images.githubusercontent.com/37491314/91118528-c0731480-e6c3-11ea-8cf7-58b799c991ff.jpg)

**Comment by lsm1103** [comment]
![image](https://user-images.githubusercontent.com/37491314/91118572-d54fa800-e6c3-11ea-896e-f2f4ad03c896.png)


asgi
![image](https://user-images.githubusercontent.com/37491314/91118609-ebf5ff00-e6c3-11ea-98b3-037056612231.png)

![image](https://user-images.githubusercontent.com/37491314/91118622-f4e6d080-e6c3-11ea-953e-660a6131bce7.png)

**Comment by lsm1103** [comment]
There will be memory overflow problems, when the recursion of key embeddings is not much

**Comment by tiangolo** [comment]
@lsm1103 if you still have problems, please follow the new issue template step by step. Without it, and all the information in it is much more difficult to help you. :coffee: 

Given that, I will close this one now.
